### PR TITLE
Link to preview of latest commit in PR body

### PR DESCRIPTION
**Pull request by jprochazk**
### Checklist
* [ ] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [ ] I've included a screenshot or gif (if applicable)

PR Build Summary: https://build.rerun.io/pr/2287

Docs preview: https://rerun.io/preview/49fbf30/docs

### CODE REVIEW

File: scripts/pr_link_docs.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+"""
+Script to generate a link to documentation preview in PRs.
+
+This is expected to be run by the `reusable_pr_link_docs.yml` GitHub workflow.
```
Comment (emilk) [inline scripts/pr_link_docs.py:6]: I think it is time to put ci-related scripts under `scripts/ci` to distinguish them from scripts meant for humans.

Other candidates:

* generate_pr_summary.py
* version_util.py
* verify_wheels.py
* generate_prerelease_pip_index.py
* check_large_files.sh
* check_large_files_allow_list.txt
* …

**Review by emilk — APPROVED** [review]
OMG YES

wow

File: scripts/pr_link_docs.py (hunk 1 above)
Comment (emilk) [reply scripts/pr_link_docs.py:6]: (but perhaps in a follow-up PR)

Comment (jprochazk) [reply scripts/pr_link_docs.py:6]: It's a small thing, but https://github.com/rerun-io/rerun/issues/2292 so we don't forget

**Pull request merged**